website/docs/guides/getting-started/getting-set-up/setting-up-databricks.md

```diff
@@ -198,34 +198,45 @@ Congratulations! At this point, you have created a Databricks account, loaded tr
 
 ### Option 2: Connect dbt Cloud and Databricks manually
 
-1. To manually setup dbt Cloud, you will need the SQL Endpoint connection information and to generate a user token. You can find your SQL endpoint connection information by going to the `Databricks UI > SQL > SQL Endpoints > Starter Endpoint > Connection details`. Save this information because you will need it later.
+#### Get endpoint and token information
 
-    <Lightbox src="/img/databricks_tutorial/images/SQL_Endpoint_Details.png" title="Databrick SQL Endpoint Connection Information" />
+To manually setup dbt Cloud, you will need the SQL Endpoint connection information and to generate a user token. You can find your SQL endpoint connection information by going to the `Databricks UI > SQL > SQL Endpoints > Starter Endpoint > Connection details`. Save this information because you will need it later.
 
-2. To generate a user token for your development credentials in dbt Cloud, click on `Settings` on the left side console (while still in the SQL part of the workspace). Click on `Personal Access Token` and provide a comment like `dbt Cloud development`. Save the token information somewhere because you will need it for the next part. 
-    <div style={{maxWidth: '400px'}}>
-    <Lightbox src="/img/databricks_tutorial/images/Generate_user_token.png" title="Generate User Token" />
-    </div>
+<Lightbox src="/img/databricks_tutorial/images/SQL_Endpoint_Details.png" title="Databrick SQL Endpoint Connection Information" />
+
+To generate a user token for your development credentials in dbt Cloud, click on `Settings` on the left side console (while still in the SQL part of the workspace). Click on `Personal Access Token` and provide a comment like `dbt Cloud development`. Save the token information somewhere because you will need it for the next part. 
+<div style={{maxWidth: '400px'}}>
+<Lightbox src="/img/databricks_tutorial/images/Generate_user_token.png" title="Generate User Token" />
+</div>
+
+#### Create a dbt Cloud account
 
 <Snippet src="tutorial-create-new-dbt-cloud-account" />
 
-3. Choose **Databricks** to setup your connection.
+#### Connect dbt Cloud to Databricks
+
+1. Choose **Databricks** to setup your connection.
     <div style={{maxWidth: '400px'}}>
```

Comment:
This should be taken care of in CSS now. I will do a final sweep to remove these in the mega PR. 
```suggestion
```